api/test/fixtures/commits.yml

```diff
@@ -10,6 +10,8 @@ rails_4ad93b:
   author: David Heinemeier Hansson
   committer_date: 2024-10-02
   author_date: 2024-10-02
+  parent_hashes: ["2fe1ca0f058c3e9ffd7fa7616e27c03aed8dba7a"]
```

Comment:
> tests do not pass, might be because it's reading the yaml fixture as JSON and it screws up. What is the best fix here?

I found this: https://thinkingeek.com/2011/06/16/fixtures-and-serialized-attributes-in-ruby-on-rails/
I think you just need to wrap `["2fe1ca0f058c3e9ffd7fa7616e27c03aed8dba7a"]` in a string? 

```
 parent_hashes: "[\"2fe1ca0f058c3e9ffd7fa7616e27c03aed8dba7a\"]
```

I think this should do the trick?
```ruby
 <%= ["2fe1ca0f058c3e9ffd7fa7616e27c03aed8dba7a"].to_json.inspect  %>
```

Reply:
Should be fixed now. Thx for the tip